src/components/Props.js

```diff
@@ -27,17 +24,34 @@ export default class Props extends React.Component {
       return name[0] !== '_' && name !== 'children' && (!defaultProps || props[name] != defaultProps[name]);
     });
 
+    const breakIntoNewLines = names.length > 3;
+    const endingSpace = this.props.singleLine ? ' ' : '';
+
     const items = [];
-    names.slice(0, 3).forEach(name => {
+    names.forEach((name, i) => {
       items.push(
-        <span key={name} style={propStyle}>
+        <span key={name}>
+          {breakIntoNewLines ? (
+            <span>
+              <br />&nbsp;&nbsp;
+            </span>
+          ) : ' '}
           <span style={propNameStyle}>{name}</span>
-          =
-          <span style={propValueStyle}><PropVal val={props[name]} /></span>
+          {/* Use implicit true: */}
+          {!props[name] || typeof props[name] !== 'boolean' && (
```

Comment:
Shall we add parenthesis `(!props[name] || typeof props[name] !== 'boolean') && (` ?
Because if someone explicitly specified `false` as the value of a boolean prop this renders it as true.